### ISSUE TITLE
Accessibility Staging Review Feedback for On This Page Component

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#21840]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh and Trevor

**Comment by 1Copenut** [comment]
👍 Thank you @joshkimux !

**Comment by CrystabelReiter** [comment]
@joshkimux @1Copenut Updated the content section of the accessibility documentation. 
@bkjohnson Will work on adding an actual component and storybook iterations

**Comment by bkjohnson** [comment]
@joshkimux the gray border is intentional, and it only appears on smaller screens:

![Peek 2021-03-24 11-13](https://user-images.githubusercontent.com/2008881/112363063-5723a600-8c92-11eb-89cf-092e98a5caa7.gif)

---

I will make the font changes to the `<h2>`.

**Comment by bkjohnson** [comment]
@CrystabelReiter said that the font size of "On this page" should be 20px, so I've made that change:

![image](https://user-images.githubusercontent.com/2008881/113359604-f97c0300-92fc-11eb-9863-7d6b417570ec.png)

**Comment by jeana-adhoc** [comment]
The component has been out in it's current stable version since 2021. This commit https://github.com/department-of-veterans-affairs/component-library/pull/169/files 5 months after the last comment removed the `<h2>` from the component due to this issue https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/252

The component is now in a nav region which makes it findable by AT users, and works as expected
